.github/workflows/main.yml

```diff
@@ -1,4 +1,4 @@
-name: GitHub Actions CI
+name: CI
```

Comment:
You need to change that name in the README batch as well, otherwise it won't be displayed correctly.

Reply:
And we should probably do the same changes in the `zycore-c` repo.

Reply:
> You need to change that name in the README file as well (GitHub actions batch), otherwise it won't be displayed correctly.

Did that now. It's 404ing for the moment -- either this refers to a different name, or it only actually applies when merged on master. Guess we'll see on merge.

Reply:
Yeah it's always referring to master. Will be good after the merge :-)